### DISCUSSION TITLE
Time for a release?

### DISCUSSION THREAD

**Discussion by jackhill** [rnpgp/rnp#2159] [General]
Hi RNP!

I see that Botan 3 support was added since the latest 0.17.0 release. Is it time for another release so that support can be made more widely available? I'm tinking about updating [GNU Guix's](https://guix.gnu.org) Botan package, and it would be nice to be able to update our RNP at the same time to make use of the new Botan.

Best,
Jack

**Comment by ni4** [comment]
Hi @jackhill ! Thanks for the interest in RNP. ETA for v0.18.0 is the late January-February, just need to close some issues which are planned to be included in this release.
